.github/workflows/build-nris.yaml

```diff
@@ -0,0 +1,33 @@
+name: Build nris
+
+on:
+  pull_request:
```

Comment:
Since we're wrapping this up I think we can remove this now :)

Reply:
Done.